### PR TITLE
Updated  documents and tutorials which supports Qiskit 1.0 and Primitives V2

### LINKED ISSUE #2114: Update documents and tutorials
### What is the expected behavior?

Since Qiskit version 1.0 is released, a lot of changes of interfaces has been implemented also in Aer. However there are still older documents and tutorials are remaining. 

- Provide detailed documentations and tutorials for V2 primitives (`qiskit_aer.primitives.EstimatorV2` and `qiskit_aer.primitives.SamplerV2`) instead of using `AerSimulator.run()` (see README of Aer)
- Remove qasm_simulator, statevector_simulator and replace with AerSimulator
- Remove other deprecate features from documents and tutorials 

Please also refer to migration guide for Qiskit 1.0 for details (https://docs.quantum.ibm.com/api/migration-guides/qiskit-1.0)

This issue will be solved by:
- #2160

### PR DESCRIPTION
**Pull request by divshacker**
### Summary
This PR Updates the documentation and tutorials according to new version of Qiskit and Primitives V2
fixes #2114 #2163 

### Details and comments
More Details can be found in the above linked issue

### CODE REVIEW

**Comment by divshacker** [comment]
Hey, @doichanj
I have updated the code used in the tutorials, Resolved every error coming due to compatibility issue. 
I will start updating the documentation now. 
If there's any thing that needs to be updated right now, Just let me know.
Thanks

File: setup.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 extras_requirements = {"dask": ["dask", "distributed"]}
 
 requirements = [
-    "qiskit>=0.45.2",
+    "qiskit>=1.1.0",
```
Comment (doichanj) [inline setup.py:25]: Is this needed for documentation?
If so, we have to discuss we can upgrade qiskit or not before merging. 
(We are planning to upgrade in Aer 0.15 release but I think it is too early to upgrade to qiskit 1.1)

Comment (divshacker) [reply setup.py:25]: This is Suggested by @kevinsung that's why I just updated. 
Anyway I don't think there should be any problem, I have made all the tutorials compatible to `qiskit 1.1`

Comment (kevinsung) [reply setup.py:25]: Perhaps I misunderstood the issue. I thought that the code updates in the tutorials would require Qiskit 1.0.

Comment (doichanj) [reply setup.py:25]: I confirm that we can merge setup.py with Qiskit 1.1 requirement

**Review by doichanj — APPROVED** [review]
thank you for updating tutorials

**Pull request merged**